### PR TITLE
FAT-1532 - mod-inn-reach: Implement API Karate tests - Locations

### PR DESCRIPTION
**Pull request by s1mpleGit**
[FAT-1532](https://issues.folio.org/browse/FAT-1532) - mod-inn-reach: Implement API Karate tests - Locations

PURPOSE:
Karate implementation is created for the following endpoints
GET /locations
POST /locations

APPROACH:
API Tests are created at  https://github.com/folio-org/folio-integration-tests
Both positive and negative cases covered
Test are shown as passed in Cucumber report.
Test run results are visible in TestRail: https://foliotest.testrail.io/index.php?/suites/overview/19
Automated regular runs are executed at https://jenkins-aws.indexdata.com/job/FOLIO_Reference_Builds/job/folio-integration/

### CODE REVIEW

**Comment by s1mpleGit** [comment]
fat1532

**Comment by s1mpleGit** [comment]
fat1532

**Review by dmtkachenko — APPROVED** [review]

File: mod-inn-reach/src/main/resources/volaris/features/inn-reach-location.feature [hunk 1]
```diff
@@ -63,6 +63,21 @@ Feature: Inn reach location
...
+    And request
+    """
+     {
+      "code": "invalid",
+      "description": "Bad test library"
+     }
+    """
+    When method POST
+    Then status 400
+
```
Comment (maryna-steshenko) [inline mod-inn-reach/src/main/resources/volaris/features/inn-reach-location.feature:80]:
```suggestion
    And match $.validationErrors[0].fieldName == 'code'
    And match $.validationErrors[0].message == 'size must be between 0 and 5'
    And match $.message == 'Validation failed'
    
```

Comment (s1mpleGit) [reply mod-inn-reach/src/main/resources/volaris/features/inn-reach-location.feature:80]: added

**Review by maryna-steshenko — APPROVED** [review]

**Pull request merged**